app/models/concerns/orchestration/tftp.rb

```diff
@@ -40,6 +40,9 @@ def generate_pxe_template
     # therefore some workaround is required to "render" the template.
     @kernel = host.operatingsystem.kernel(host.arch)
     @initrd = host.operatingsystem.initrd(host.arch)
+    if host.operatingsystem.pxe_type == 'yast'
+      @mediapath = host.operatingsystem.mediumpath(host)
+    end
```

Comment:
In the first version I did check if mediatpath is defined, but `test_generate_pxe_template_for_localboot` did fail then and I didn't look further... In general it's an old, known problem that only SuSE does need the mediapath in its PXELinux template.